chat-services/mattermost-gitlab.yaml

```diff
@@ -0,0 +1,6 @@
+services:
+- hash: f2cf7f90f65214143db61c42fb06d0dda3eebaa6
+  hash_length: 7
+  name: chat-integrations-gitlab
```

Comment:
this should be `chat-integrations-github`

Reply:
fixed the typo